forem/forem
# Multiple reactions - Reading list toggle issue fix

## What type of PR is this? (check all applicable)


## Description

If multiple reactions is enabled and you click on `Bookmark` on any article, the heart icon also toggles. Check below video. 
This PR fixes that issue.

## Related Tickets & Documents



- Related Issue #19008

## QA Instructions, Screenshots, Recordings

**How to test?**
1. Enable the `multiple_reactions` FeatureFlag at `/admin/feature_flags/features`
2. View an article
3. Click on `Bookmark` / `Reading List`
4. On main branch you will notice that the `heart+` icon toggles too and on this branch you will notice that it works correctly.

**Before**

https://user-images.githubusercontent.com/9396084/215767396-73188b43-8f80-4dde-a45e-1122cc6c4704.mov 

**After**

https://user-images.githubusercontent.com/9396084/215767494-901a0440-0c81-430f-87a2-565b83204b36.mov

### UI accessibility concerns?

No

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

      replace this line with details on why this change doesn't need to be
      shared_
